cpp/ps0000_sol.cpp

```diff
@@ -0,0 +1,8 @@
+#include <iostream> // 2/2
+using namespace std;
+int main() // 1/1
+{
+  cout << "Hello, World!\n"; // 4/4
+  return 0; // 0/1
+}
+// great work 😎 8/10 ⭐
```

Comment:
- [x] most of code errors and bugs are fixed
- [x] file is in the right path
- [x] +1 mark for your first commit